tests/functional/firefox/test_sync.py

```diff
@@ -23,6 +23,13 @@ def test_android_download_button_displayed(base_url, selenium):
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
+def test_ios_download_button_displayed(base_url, selenium):
```

Comment:
Usually it's better to have separate tests for things have that separate interactions, but I think here it's fine to combine them as we're just checking the display of two buttons in the same region.